pages/docs/integrations/grouparoo-customerio.mdx

```diff
@@ -0,0 +1,150 @@
+    </tr>
+    <tr>
+      <td>
+        <strong>Site ID</strong> (required)
+      </td>
+      <td>Customer.io Tracking API site ID</td>
+    </tr>
+    <tr>
+      <td>
+        <strong>Tracking API Key</strong> (required)
+      </td>
+      <td>Customer.io Tracking API key</td>
+    </tr>
+    <tr>
+      <td>
+        <strong>App API Key</strong> (required)
+      </td>
+      <td>Customer.io App API key</td>
+    </tr>
+  </tablebody>
+</table>
+
+## Customer.io Export Customers
+
+This Destination allows you to sync your Grouparoo Records to Customers in Customer.io.
+
+Companies use Customer.io to orchestrate data-driven communications through email, pugs notifications, and SMS messaging.
+
+When you [create a Destination](https://www.grouparoo.com/docs/config/destinations), you make choices about what data to sync.
+
+### Sync Modes
+
+- **Sync**: Add, update, and remove Customer.io Customers as needed. This option looks for new records, changes to existing records, and deletions.
+- **Additive**: Add and update Customer.io Customers as needed, but do not remove anybody. This option looks for new Customers to add to Customer.io and changes to existing Customers, but does not keep track of deletions.
+- **Enrich**: Only update those Customers that already exist in Customer.io. Do not add or remove Customers.
+
+### Destination Options
+
+Some types of Destinations require further clarification about how data should sync. In the case of Customer.io, there are no further Options to configure before moving on to mapping Record Data.
+
+### Record Data
+
+Mapping Grouparoo Record Properties to Customer.io Attributes is the key to syncing data to Customer.io Customers.
+
+<ImageInBrowserFrame
+  src="/images/docs/integrations/customerio/export-customers/properties.png"
+  alt="Customer.io Export Customers Properties"
+  width={733}
+  height={378}
+/>
+
+<br />
+
+#### Required Properties
+
+You must map a Grouparoo Property to the **customer_id **Attribute in Customer.io. Choose the Grouparoo Property that uniquely identifies each individual customer’s record from the dropdown menu.
+
+- **customer_id:** A unique identifier that relates a Customer record to a Record in Grouparoo .
+
+#### Optional Properties
+
+In addition to being able to map required properties and properties already known to exist in Customer.io, this plugin offers you the ability to add new information to Customer.io by selecting a Grouparoo property and mapping to a new Attribute.
+
+<ImageInBrowserFrame
+  src="/images/docs/integrations/customerio/export-customers/optional.png"
+  alt="Customer.io Export Customers Optional Properties"
+  width={733}
+  height={162}
+/>
+
+<br />
+
+### Group Data
+
+Any groups you have made in Grouparoo would be available to use as a Customer.io Group Attribute. The Group Attributes you assign will help marketers engage with customers more effectively by applying meaningful segmenting to your company’s audience.
```

Comment:
"would be". Should it be "can be" or "will be" ?

Reply:
"Will be" is probably more clear. I'll update that.